jbmc/regression/jbmc-strings/CharacterToChars/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+C.class
+--function C.test --cp `../../../../scripts/format_classpath.sh . ../../../lib/java-models-library/target/core-models.jar`
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
```

Comment:
Test will pass even if `toChars` is not implemented. because unreachable evaluates to true. Could we add an assertion that we expect to fail?